homeassistant/config.py

```diff
@@ -343,7 +343,24 @@ def _load_hass_yaml_config() -> Dict:
                 "Config file not found in: {}".format(hass.config.config_dir))
         return load_yaml_config_file(path)
 
-    return await hass.async_add_executor_job(_load_hass_yaml_config)
+    config = await hass.async_add_executor_job(_load_hass_yaml_config)
+    await async_hass_config_merge(hass, config)
```

Comment:
This is inefficient going to executor twice, we should just call the merge config from the executor job that we are defining above here

Reply:
:+1: